openhabian-setup.sh

```diff
@@ -1060,39 +1060,38 @@ HABian GitHub issue."
   fi
 
   /bin/mkdir -p ${FIND_DSTDIR}
   /usr/bin/wget -O ${FIND_TMP} ${FIND_SRC}
   /usr/bin/wget -O ${CLIENT_TMP} ${CLIENT_SRC}
   /usr/bin/unzip ${FIND_TMP} findserver -d /usr/sbin
   /usr/bin/unzip ${FIND_TMP} fingerprint -d /usr/sbin
-  /usr/bin/unzip ${FIND_TMP} static\* -d ${FIND_DSTDIR}
+  /usr/bin/unzip ${FIND_TMP} -d ${FIND_DSTDIR}
 
   FINDSERVER=$(whiptail --title "FIND Setup" --inputbox "Enter hostname that your FIND server will be listening to:" 15 80 localhost 3>&1 1>&2 2>&3)
   FINDPORT=$(whiptail --title "FIND Setup" --inputbox "Enter port no. that you want to run FIND server on:" 15 80 8003 3>&1 1>&2 2>&3)
-  MQTTSERVER=$(whiptail --title "FIND Setup" --inputbox "Enter hostname that your MQTT server is running on:" 15 80 localhost 3>&1 1>&2 2>&3)
-  MQTTPORT=$(whiptail --title "FIND Setup" --inputbox "Enter port no. that your MQTT server is running on:" 15 80 1883 3>&1 1>&2 2>&3)
+  MQTTSERVER=$(whiptail --title "FIND Setup" --inputbox "Enter hostname that your MQTT broker is running on:" 15 80 localhost 3>&1 1>&2 2>&3)
+  MQTTPORT=$(whiptail --title "FIND Setup" --inputbox "Enter port no. that your MQTT broker is running on:" 15 80 1883 3>&1 1>&2 2>&3)
 
   if [ -f ${MOSQUITTO_PASSWD} ]; then
-    FINDADMIN=$(whiptail --title "findserver MQTT Setup" --passwordbox "Enter a username for FIND to use as the admin user on your MQTT ser
-ver:" 15 80 3>&1 1>&2 2>&3)
-    FINDADMINPASS=$(whiptail --title "findserver MQTT Setup" --passwordbox "Enter a password for the FIND admin user on your MQTT server:"
- 15 80 3>&1 1>&2 2>&3)
+    FINDADMIN=$(whiptail --title "findserver MQTT Setup" --inputbox "Enter a username for FIND to use as the admin user on your MQTT broker:" 15 80 $DEFAULTFINDUSER 3>&1 1>&2 2>&3)
+    FINDADMINPASS=$(whiptail --title "findserver MQTT Setup" --passwordbox "Enter a password for the FIND admin user on your MQTT broker:" 15 80 $DEFAULTFINDPASS 3>&1 1>&2 2>&3)
     cond_redirect /usr/bin/mosquitto_passwd -b ${MOSQUITTO_PASSWD} ${FINDADMIN} ${FINDADMINPASS} || FAILED=1
     if [ $? -ne 0 ]; then echo "FAILED"; exit 1; fi
   fi
   /bin/sed -e "s|%MQTTSERVER|${MQTTSERVER}|g" -e "s|%MQTTPORT|${MQTTPORT}|g" -e "s|%FINDADMIN|${FINDADMIN}|g" -e "s|%FINDADMINPASS|${FINDADMIN_PASS}|g" -e "s|%FINDPORT|${FINDPORT}|g" -e "s|%FINDSERVER|${FINDSERVER}|g" ${SCRIPTDIR}/includes/findserver.service >${FIND_SYSTEMCTL}
   /bin/sed -e "s|%MQTTSERVER|${MQTTSERVER}|g" -e "s|%MQTTPORT|${MQTTPORT}|g" -e "s|%FINDPORT|${FINDPORT}|g" -e "s|%FINDSERVER|${FINDSERVER}|g" ${SCRIPTDIR}/includes/findserver >${FIND_DEFAULT}
 
 
-  /bin/rm -f ${FIND_TMP} ${CLIENT_TMP}
+  /bin/rm -f ${FIND_TMP} ${CLIENT_TMP} ${FIND_DSTDIR}/findserver
```

Comment:
Also "fingerprint" ?

Reply:
no

Reply:
But then it's available twice.
May I suggest something instead of the above. That is how I'd do something like this:

```shell
unzip ${FIND_TMP} -d ${FIND_DSTDIR}
ln -s ${FIND_DSTDIR}/findserver /usr/sbin/findserver
ln -s ${FIND_DSTDIR}/fingerprint /usr/sbin/fingerprint
```

Reply:
no, fingerprint isn't contained in the archive.
Then again, $FIND_TMP  in the unzip line is bad, it must read $CLIENT_TMP
Ok, will change that and do a ln for the findserver binary

Reply:
Ah sorry, didn't see that detail. Now remove the `/bin/rm -f ${FIND_DSTDIR}/findserver` and we are good to go ;)

Reply:
here you are